### DISCUSSION TITLE
Image not showing up in NavBar

### DISCUSSION THREAD

**Discussion by TarynL** [nss-day-cohort-47/C47-Discussions#34] [Help Me] [answered]
Name: Taryn Lytle

Have you searched the other help tickets?
There are no tickets for my particular problem.

Describe your issue:
In the giffygram project, there are 2 images we need to upload into the images folder. One of them is a SVG and I am having problems getting it to pop up in both VS Code and the dom.

List any error messages that appear:
Failed to load resource: the server responded with a status of 404 (Not Found)

What have you googled?
https://stackoverflow.com/questions/41195669/images-in-svg-image-tags-not-showing-up-in-chrome-but-displays-locally

https://forum.webflow.com/t/svg-file-not-showing/93954

What have you tried? Tell us everything you have tried.
I have tried saving it directly to my desktop then importing it into VS Code.
I have tried changing the height and width.
Attempted to change the format but could not figure it out.

Copy/paste link TO THE FILE on your branch or in your repo that you are having issues with:
https://github.com/TarynL/GiffyGram.git

**Comment by Chadwell-Clark** [comment]
Hey Taryn I tried to look at your code but I don't think your latest has been pushed to github. It could be that the path is off.  I think you may need to add "../images/".
I think it was written as "./images/" in the lesson

**Reply by TarynL** [reply]
Hey Chad! I just pushed my latest branch. Are you talking about changing that in the NavBar.js? I tried that and didn't see it still. The PB img shows just fine. The image in VS only shows code, I can't even get it to pop up properly there. I'm assuming it has to do with formatting but that is out of my wheel house

**Reply by Chadwell-Clark** [reply]
I think your fountain pen image name needs to be changed in your images folder. The image is a copy and has (1) at the end of the name

**Reply by TarynL** [reply]
I caught that a bit ago. Already updated. Still nothing. Keep the ideas coming:)

Your help is very much appreciated!

**Reply by TarynL** [reply]
I just ended up using an online format converter to change it to a jpg and that seemed to do the trick. I feel like that was cheating a bit so I would love to figure out a better solution.

**Reply by BryanNilsen** [reply]
I believe @Chadwell-Clark is correct in that the path to the image is/was your issue... including the file name of the image was incorrect -- the file you're trying to access in the navbar is `fountain-pen.svg`, whereas you had `fountain-pen (1).svg`

The chapter has been updated to reflect the correct directory structure to implement.
The `images` directory should be in the `src` directory, not in the `scripts` directory. That should resolve the path issues you're having with the image tags. 

Let us know how you resolve it (other than the online format converter) and select one of these answers to complete the discussion. Thanks!! This is great!!

**Comment by Chadwell-Clark** [comment]
I will take a look here in a few minutes.Sent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: TarynL <notifications@github.com> Date: 3/8/21  6:45 PM  (GMT-06:00) To: nss-day-cohort-47/C47-Discussions <C47-Discussions@noreply.github.com> Cc: Chadwell Clark <wcclark23@comcast.net>, Comment <comment@noreply.github.com> Subject: Re: [nss-day-cohort-47/C47-Discussions] Image not showing up in NavBar (#34) 
I just ended up using an online format converter to change it to a jpg and that seemed to do the trick. I feel like that was cheating a bit so I would love to figure out a better solution.

**Comment by Chadwell-Clark** [comment]
I took a look at it and renaming the fountain-pen image in the images folder from "fountain-pen (1).svg" to "fountain-pen.svg" fixed it when I ran it. There could be another issue but I am not seeing it.
You should rename the app to PuggyGram! Looks great.

**Reply by pour-theoil** [reply]
Chad, I had a look as well and it seems that works for me too. Looking really good! 

I noticed that in your index.html you have the text "My List" that has been written over by your nav bar was that on purpose?

**Reply by Chadwell-Clark** [reply]
That would be the result of using = rather than += when inserting it on the DOM.

**Reply by Chadwell-Clark** [reply]
Or css causing it to be hidden 😉

**Comment by TarynL** [comment, marked as answer]
I pulled my images folder out of styles and into src then updated my NavBar.js with the fountain-pen.svg image and updated the path. It works perfectly. 

Additionally, I removed the fixed positions in both the footer and navigation css. Its all coming together. Thank you all for your help.  A special shoutout to Chad and all the pugs out there!!
